torch/_inductor/codegen/wrapper.py

```diff
@@ -1058,7 +1058,7 @@ def generate_end(self, result):
 
         result.writeline("'''\n)")
         # get the hash of the wrapper code to name the extension
-        wrapper_call_hash = codecache.code_hash(self.wrapper_call.getvalue())
+        wrapper_call_hash = codecache.code_hash(result.getvalue())
```

Comment:
@jgong5 @eellison, I re-open this PR to fix ```cpp_wrapper``` issue which using all code as hash rather than only ```wrapper_call``` code(there has case that ```wrapper_call```  code is same for two models, but whole code is not same)